CMakeLists.txt

```diff
@@ -336,10 +336,6 @@ message(STATUS "Using security hardening compiler flags: ${HARDENING_CFLAGS}")
 message(STATUS "Using security hardening linker flags: ${HARDENING_LDFLAGS}")
 message(STATUS "Using security hardening compiler definitions: ${HARDENING_DEFINITIONS}")
 
```

Comment:
Add a comment that these flags are applied through `open3d_set_global_properties`

Reply:
Done. Moved these to `cmake/Open3DSetGlobalProperties.cmake`.